### ISSUE TITLE
[a11y Design Review] Ask VA Team Phase 1 - Midpoint Prep (Nov. 14)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#68938]
## Auditor(s)

@eli-oat 
@coforma-jamie 

## About the AVA Team

<details><summary>Toggle to view AVA team info.</summary>

- **Team name:** Ask VA (AVA)
- **OCTO-DE product owner:** @clantosswett
- **Product manager:** @ruchishah29
- **Designer:** @t-michaud @beckyphung @tygindraux
- **FE engineer:**
- **Accessibility specialist:**
- **Content writer:**
- **Product/team Slack channel**: [#ask-va-public](https://dsva.slack.com/archives/C05A2F6DEAE)
- **Related Epic from the Collaboration Cycle:** #60696
</details>

## Important Notes
- **Focus:** Phase 1 Goals (refer to background information in this ticket)
- **Midpoint Review:** November 14, 2023 at 11:30 a.m. Eastern (source Collab Cycle Ticket)

## Background Information
Overview information collected during the kickoff meeting held in July of 2023.
<details><summary>Toggle for a brief overview of initial a11y considerations</summary>

### Some Initial A11y Considerations:
- Mobile Focus (per Martha)
- Removing tables and reserving them for data only.
- One-item per page for cognition optimization.
- SR research (Jan. 2024)
</details>

<details><summary>Toggle to access background info. on this project</summary>

### Existing Product
- **This is an existing product:**
- There are apparently a lot of issues with the tool as it exists now.
### Initial goals
- "**Quickly lift and shift (with a facelift)** on Ask Va to bring the functionality over to [Va.gov](http://va.gov/) and forward thinking about it more strategically as a larger support ecosystem within the va and [va.gov](http://va.gov/)."
    - "**Our Phase 1** will have brought the current Ask VA up to [VA.gov](http://va.gov/) design and **accessibility standards**, updated the content to **make it more intuitive for our submitters**, and sunset the existing front-end portal of AVA."
    - **AKA P1 Focus:**
        - "**Phase 1 - exclusively focusing on** the web form experience, as well as the ability to look up your inquiry and see the status of it."
### Longer-Term Goals
- **Accurately answer submitters' questions:**
   - "Submitters should feel it is easy and intuitive to submit their issues and receive adequate updates and information that results in a final resolution.
  - Ultimately, we want AVA to feel part of a cohesive and submitter-centered approach to support."
### Collab Cycle
- We shared a slew of resources in the doc and they implemented our recommendations prior to design intent.
- **Midpoint Review:**` Nov. 14 at 11:30 a.m. Eastern` ([via Zoom](https://oddball-io.zoom.us/j/81600041855?pwd=ejZrVW9jNXp4Qmx4Mzc0VjNCVHREZz09))
</details>

<details><summary>Toggle to review Governance Design Intent Accessibility Feedback</summary>

## Governance Feedback

> I think you've got the right priorities, and you're already working with the CAIA folks. Looking forward to seeing you at Midpoint Review! Very much appreciate the one-thing-per-page approach!  [Feedback from Brian 7/31/2023](https://github.com/department-of-veterans-affairs/va.gov-team/issues/62901)

### Consider
- Right now, there's some inconsistency between different steps in the workflow where some have a descriptive heading and/or fieldset/legend, while others just have the input beneath the step indicator. There are a couple of other teams who have been working on form flows in which each page has a unique H3 on each page that either serves as a statement of the question on the page or summarizes the collection of questions. This serves to:
    - Have consistent visual weight moving through the workflow.
    - Provide a consistent target for receiving keyboard focus that's unique for each page, so screen reader users get immediate feedback on their progress even when there are multiple pages in a single step.
This isn't something required or even something super common on VA.gov, but it's a nice practice to follow if you're able. I'd encourage you to reach out to Josh Kim and Robin Garrison for their thoughts/examples of implementation.

</details>

## Prototype Links
<details><summary>Toggle to View Prototype Links</summary>

### Test with Sketch Prototypes
**Artifacts taken from the Collab Cycle Ticket `#62304` on 11/1/2023**
-  **Finalized design mockup:** [Sketch - Ask VA Form - C - VA Health Care [WIP]](https://sketch.com/s/6a75d0a2-e484-4f1e-8675-ad5747a1a871)
-  **Specific page to review:** [V3 - C - VA Health Care Flow](https://www.sketch.com/s/6a75d0a2-e484-4f1e-8675-ad5747a1a871/p/4E7790A9-059D-4A3B-A184-45F1E9434CDB/canvas)
</details>

## Design Review Tasks for CAIA A11ys 

- The following are the areas that we traditionally review for the CAIA A11y Design Review.
- _Based on the development of the prototype, some of the following may or may not be applicable._ 
- @eli-oat you mentioned a consultation approach, so feel free to amend this ticket and its tasks as needed to accommodate that process you are developing. 

```[tasklist]
### CAIA a11y Comms
- [x] Connect with the Content Team 11/2/2023
- [x] Ask VA Team intro and initial reach out 11/2/2023 c/o Jamie
- [x] Large Sketch file R&D with Ask VA team to get it to a manageable loadable size for review taking place 11/2/2023
- [ ] Assist team with prep for Midpoint @eli-oat via consultation approach
- [ ] **TBD** - Conduct Full Design Review (see traditional test areas task list below)
- [ ] **TBD** - Share Midpoint Prep Feedback Deliverable with the product team prior to Midpoint - scheduled for 11/14/2023 (see deliverable tasks list below)
```
```[tasklist]
### Traditional Design Review Testing
- [ ] [Color and contrast checks](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#use-of-color)
- [ ] **Color and contrast checks:** WCAG 2.1 AA
- [ ] **Color and contrast checks:**  WCAG 2.1 AAA
- [ ] **Color and contrast checks:**  [Who Can Use](https://www.whocanuse.com/?b=ecece5&c=000000&f=20&s=)
- [ ] **Color and contrast checks:**  High Contrast Mode
- [ ] [**Keyboard navigation**](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#keyboard-nav) Mobile & Desktop
- [ ] **Keyboard Navigation**: Annotations for Tab order, reflow and reading order  Mobile & Desktop **(if needed)**
- [ ] **Accessible Name Annotations**: [Alt attributes for images](https://webaim.org/techniques/alttext/) Mobile & Desktop
- [ ] **Accessible Name Annotations**: Text alternative for complex images such as charts graphs maps etc Mobile & Desktop
- [ ] **HTML Tags:** [Page Title](https://accessibility.18f.gov/page-titles/) Mobile & Desktop
- [ ] **HTML Tags:** [Heading order](https://accessibility.18f.gov/headings/) Mobile & Desktop
- [ ] **HTML Tags:** Annotations (if needed)
- [ ] **HTML Tags:** Semantic Consideration for paragraphs ordered lists numbering Mobile & Desktop
- [ ] **HTML Tags:** [Tables (heading rows)](https://design.va.gov/components/table) Mobile & Desktop
- [ ] **Size of clickable areas**: Mobile & Desktop
- [x] [**Components**](https://design.va.gov/components/) follow VA design system guidance. (As time allows)
- [x] [**Patterns**](https://design.va.gov/patterns/) follow VA design system guidance. (As time allows)
```
```[tasklist]
### Deliverable from CAIA a11ys
- [x] **Create** markdown draft of the [Design Review Deliverable Template](https://github.com/department-of-veterans-affairs/va.gov-team/edit/master/teams/CAIA/accessibility/templates/Design%20review%20template.md) to create the final deliverable summary and document findings from the design review.
- [x] **Create** a folder for this team via [Teams > CAIA > Accessibility > Deliverables](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/teams/CAIA/accessibility/Deliverables)
- [x] **Save** the final deliverable in this new folder
- [x] **Link** to the deliverable in this ticket [DELIVERABLE](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/Deliverables/Ask%20VA/2023-11-02-Ask-VA-design-review-notes.md)
- [x] **Share** the link in the Slack Channel for the team (found in the about section, at the top of this ticket)
```
